.claude/hooks/pre_git_quality_gates.py

```
@@ -108,58 +114,76 @@
-                "stderr": result.stderr,
-                "returncode": result.returncode
-            })
-            print(f"  ❌ 失敗 (exit code: {result.returncode})", file=sys.stderr)
+            # returncode 127 またはコマンドが見つからないエラーはスキップ
+            is_tool_missing = (
+                result.returncode == 127
+                or "No such file or directory" in result.stderr
+                or "command not found" in result.stderr
+            )
+            # 出力が実質的に空（ヘッダーのみ）の場合もツール/環境の問題としてスキップ
+            stdout_lines = [l for l in result.stdout.strip().split('\n') if l.strip() and not l.startswith('━')]
```

Ruff (0.15.0)

[error] Ambiguous variable name: `l`

(E741)